crd-generator/apt/README.md

```diff
@@ -0,0 +1,52 @@
+# CRD Generator Annotation Processor
+_Deprecated since 7.0.0_
+
+## Usage
+
+with Maven:
+
+```xml
+<dependency>
+    <groupId>io.fabric8</groupId>
+    <artifactId>crd-generator-apt</artifactId>
+    <scope>provided</scope>
+</dependency>
+```
+
+with Gradle:
+
+```groovy
+dependencies {
+    annotationProcessor 'io.fabric8:crd-generator-apt:<version>'
+}
+```
+
+### Experimental
+
+#### Generate CRDs in parallel
+
+It's possible to speed up the CRDs generation by using parallel computation.
+Please note that this feature is experimental, and it may lead to unexpected results.
+
+To enable it, you need to set the `io.fabric8.crd.generator.parallel` property to `true` in the processor.
+
+with Maven:
+
+```xml
+<plugin>
+  <artifactId>maven-compiler-plugin</artifactId>
+  <configuration>
+    <compilerArgs>
+      <arg>-Aio.fabric8.crd.generator.parallel=true</arg>
```

Comment:
Speaking of which, I don't recall if how to enable this with the maven plugin is documented?

Reply:
With the new maven plugin and CLI parallel is enabled by default. But you can disable it via normal configuration:

https://github.com/fabric8io/kubernetes-client/tree/main/crd-generator/cli#usage
https://github.com/fabric8io/kubernetes-client/blob/main/crd-generator/maven-plugin/README.md#more-configuration-parameters

The section moved in this PR does only apply to the old annotation processor and api-v1.